content/en/database_monitoring/guide/aurora_autodiscovery.md

```diff
@@ -0,0 +1,240 @@
+
+The beta version can be found [here][9]. You can use the Agent installation script to install the correct version by running the following command:
+
+```bash
+DD_API_KEY=${API_KEY} DD_SITE="datadoghq.com" \
+DD_AGENT_DIST_CHANNEL=beta DD_AGENT_MAJOR_VERSION=7 \
+DD_AGENT_MINOR_VERSION=52.0~dbm~aurora~autodiscovery~beta~0.3-1 \
+bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+```
+
+For more information on installing the Datadog Agent, please read the following [documentation][10].
+
+### AWS permissions
+
+It's required that the Agent has permission to run `rds:DescribeDBClusters` and `rds:DescribeDBInstances` in your AWS account. The recommended way to do this, is to attach an IAM role policy to the EC2 instance where the Agent is running.
+
+An example policy that grants the required permissions is:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "rds:DescribeDBClusters",
+        "rds:DescribeDBInstances"
+      ],
+      "Resource": "arn:aws:rds:<region>:<account>:cluster:<name>"
+    }
+  ]
+}
+```
+
+You can also attach the [AmazonRDSReadOnlyAccess][3] policy.
+
+### Configure aurora
+
+By default, the listener discovers all aurora clusters in the account and region where the Agent is running that have the `datadoghq.com/scrape:true` tag applied. You can also configure the Agent to discover clusters with specific tags.
+
+The tags must be applied to the DB cluster (Role: `Regional clutser`). For more information on tagging RDS resources, see the [AWS documentation][7].
+
+### Configure the Datadog Agent
+
+This utilizes an Agent service listener, responsible for discovering all database host endpoints in an aurora cluster and forwarding discovered endpoints to the existing Agent check scheduling pipeline. You can configure the listener in the `datadog.yaml` file:
+
+```yaml
+database_monitoring:
+  autodiscovery:
+    aurora:
+      enabled: true
+```
+
+**Note**: The Agent only discovers Aurora instances running in the same region as the Agent. In order to discover the region, the Agent uses [IMDS (Instance Metadata Service)][8] to determine the region where it is running. If your Ec2 instance requires `IMDSv2`, you need to configure the Agent to use `IMDSv2` by setting `ec2_prefer_imdsv2: true` in the `datadog.yaml`.
+
+Specify IMDSv2 in the `datadog.yaml` file:
+
+```yaml
+ec2_prefer_imdsv2: true
+database_monitoring:
+  autodiscovery:
+    aurora:
+      enabled: true
+```
+
+By default, the listener discovers all aurora clusters in the account and region where the Agent is running that have the `datadoghq.com/scrape:true` tag applied. You can also configure the listener to discover clusters with specific tags.
+
+Specify custom tags for the aurora cluster discovery in the `datadog.yaml` file:
+
+```yaml
+database_monitoring:
+  autodiscovery:
+    aurora:
+      enabled: true
+      tags:
+        - "my-cluster-tag"
```

Comment:
```suggestion
        - "my-cluster-tag-key:value"
```
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏just to make it clear it needs to follow the `<KEY>:<VALUE>` pattern